lib/experimental/html/class-wp-html-attribute-token.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * HTML Walker: Attribute token structure class.
+ *
+ * @package WordPress
+ * @subpackage HTML
+ * @since 6.1.0
+ */
+
+/**
+ * Data structure for the attribute token that allows to drastically improve performance.
```

Comment:
@adamziel and @dmsnell, is the note about the performance implications valid? Should we also include the same note for `WP_Class_Name_Update` and `WP_Text_Replacement`? I know you discussed that extensively, but I'm not sure which part made the biggest difference.

Reply:
It is valid, I think in @dmsnell's testing it reduced memory usage by like 90% compared to `array()` which sounds almost unbelievable.

Reply:
Yes we can update these comments across the board. `array()` was surprisingly _terribly_ inefficient with memory use.